src/schema/v2/artist/helpers.ts

```diff
@@ -163,6 +164,11 @@ export const ARTIST_INSIGHT_MAPPING: Record<
         count === 1 ? "a top industry award" : `${count} top industry awards`
       }`,
   },
+  FOUNDATIONS: {
+    getDescription: () => null,
+    getEntities: (artist) => splitEntities(artist.foundations) || null,
```

Comment:
do you need the `|| null`? I see the other kinds don't need it when calling `splitEntities`

Reply:
yeps